rfc/0029-community-amendment-1.md

```diff
@@ -0,0 +1,304 @@
+events and generally engage with the community when possible.
+
+## Keyholders
+
+While Quilt is a community project with concerns split across different teams, Discord (and some other platforms) is
+designed with the core idea of a single community leader or owner in mind. For that reason, platforms that cannot be
+run explicitly under team leadership alone must be "owned" by a keyholder.
+
+A keyholder is a democratically elected member of the community that otherwise has no direct involvement or stake
+in the management of the community. By electing an otherwise unrelated party to be a keyholder, it's hoped that the
+Quilt community spaces can avoid the issue of a community leader gone rogue - it should prevent an abuse of power from
+giving someone total control over any community space, at least as far as is possible.
+
+After they've been elected, a keyholder does not need to be otherwise active in the community. All that's required of
+them is that they're easily contactable (within reason), and that they're prepared to act in the event that something
+happens that they will need to take care of.
+
+Keyholder candidates should:
+
+* Be able to show that they have no conflicting interests or personal wish to be in power
+* Be known to a majority of the Community Team
+* Have a clean, positive reputation in the Quilt community and wider modding community
+* Have some accreditation or other guarantee of reputation in the wider modding community
+* Regarding Discord specifically, they should be prepared to act as a liaison with the Discord team and to represent
+  the Quilt community officially when interacting with them, including in the event of a partnership or working
+  relationship with Discord
+* Understand what's involved with keyholding, and be ready to step in when needed
+
+Keyholders are not members of the Community Team, and they are not required to be obvious in the communities they
+keyhold for - but they are free to make themselves known if they wish. They're also welcome to oversee the community 
+team in their actions and processes, and provide opinions and feedback if they wish.
+
+## Basic Responsibilities
+
+All members of the Community Team have the following responsibilities:
+
+* To represent the Quilt project in an honest and respectful manner, and serve as an example of good behaviour in
+  community spaces
+* To participate in the processes that shape the Community Team, including voting on new team members and structural
+  changes to community spaces (unless they're a Trainee Community Moderator - see below)
+* To engage with the community authentically, being present and visible in the community when able and appropriate
+* To be supportive of other Community Team members, providing insight, advice and support when able and appropriate
+* To help with the planning and running of community events and projects, when able and appropriate
+
```

Comment:
Suggestion to include here what tools/powers are available to the 'basic' Team. I propose only what is strictly essential for the position. For example, moderation powers will not be required, but in order to participate in electing new team members, ModMail viewing permission at *least* will be necessary.

Reply:
The way it currently works is that moving a modmail thread out of the "Backlog" category and into one of the others will prompt you to sync perms. If you decline this, only mods and managers can see it - if you accept, admins can also see it.

Of course, once this RFC is in, we'll need to reorg the community team role structure a bit - but it's the same kind of idea.